src/lib/CMakeLists.txt

```diff
@@ -58,8 +60,8 @@ add_subdirectory(perf)
 add_subdirectory(pid)
 add_subdirectory(rc)
 add_subdirectory(systemlib)
+add_subdirectory(tecs)
 add_subdirectory(terrain_estimation)
 add_subdirectory(tunes)
 add_subdirectory(version)
 add_subdirectory(weather_vane)
-add_subdirectory(mag_compensation)
```

Comment:
This is not directly linked to the rest of this PR, right?

Reply:
I sorted the list.